packages/@tinacms/toolkit/src/packages/core/media.ts

```diff
@@ -272,3 +272,22 @@ export class MediaListError extends Error {
     this.docsLink = config.docsLink
   }
 }
+
+export const E_UNAUTHORIZED = new MediaListError({
```

Comment:
These error messages are kind of a mixture between Tina Cloud and Cloudinary messages (the docs links will obviously get updated when we have them)